scripts/archive-images-lists.sh

```diff
@@ -28,7 +39,14 @@ for prev_ver in $(echo "$previous_versions"); do
 
   mkdir "$WORKING_DIR"/"$prev_ver"
 
-  curl -fL https://releases.rancher.com/harvester/"$prev_ver"/image-lists.tar.gz -o "$WORKING_DIR"/image-lists.tar.gz || ret=$?
+  image_lists_url=https://releases.rancher.com/harvester/"$prev_ver"/image-lists.tar.gz
+  # arch-aware image is available after v1.3.0
+  if [ "$(version_compare "$prev_ver" "v1.3.0")"  -eq "1" ]; then
```

Comment:
Though it works, we usually use `-eq` to compare integers. It's better to follow the rules. Thank you.

Reply:
oops, yes it should be integer, my bad :(